### ISSUE TITLE
How do you run it?

### ISSUE THREAD

**Issue by lance-edmodo** [PalmerAL/min#115]
I downloaded and installed the .deb on Linux Mint (Ubuntu variant).

How do you start the browser?

**Comment by burnoutberni** [comment]
How did you install it? Did you ```dpkg -i /path/to/Min.deb```?
If yes it should show up like any other program in your desktop environment ànd you should be able to start it on the command line with ```Min``` (case-sensitive).

**Comment by lance-edmodo** [comment]
This does not happen.

What is the command line to start it? Where are the files installed?

**Comment by ljoli** [comment]
@lance-edmodo  Yes , You are allright. when we install deb package, we have
```
{ % }: sudo dpkg -i min_1.2.0_i386.deb                                                                                                                                                                                       29.Apr.16 10.51
dpkg: warning: parsing file '/var/lib/dpkg/available' near line 580187 package 'min':
 missing description
dpkg: warning: parsing file '/var/lib/dpkg/available' near line 580187 package 'min':
 missing maintainer
Selecting previously unselected package min.
(Reading database ... 318816 files and directories currently installed.)
Preparing to unpack min_1.2.0_i386.deb ...
Unpacking min (1.2.0-1) ...

Setting up min (1.2.0-1) ...
Processing triggers for desktop-file-utils (0.22-1ubuntu1) ...
Processing triggers for gnome-menus (3.10.1-0ubuntu2) ...
Processing triggers for mime-support (3.54ubuntu1.1) ...
```
And when we search binary 

```
{ % }: which Min                                                                                                                                                         
Min not found
```
I do that with min_1.2.0_i386.deb on UBUNTU 14.04LTS

**Comment by PalmerAL** [comment]
I can reproduce this.

**Comment by emersion** [comment]
I can reproduce. The file `/usr/bin/min` is a dead symbolic link to `/usr/share/min/min`:
```
/usr/bin$ ls -l | grep min
lrwxrwxrwx 1    501 staff         16 avril 28 06:48 min -> ../share/min/min
```
The file should point to `../share/min/Min` instead.

**Comment by PalmerAL** [comment]
Thanks, that's really helpful!

This is fixed in aba88faa40eb91169158e78a1307eb5e74bffdde. I'll publish a 1.2.1 release soon that has this fix included.

**Comment by PalmerAL** [comment]
1.2.1 has been released.
